build-system/babel-plugins/babel-plugin-is_dev-constant-transformer/index.js

```diff
@@ -0,0 +1,40 @@
+/**
+ * Copyright 2018 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+  * Changes the values of IS_DEV to false and IS_MINIFIED to true.
+  * The above said variables are in src/mode.js file.
+  * @param {Object} babelTypes
+  */
+module.exports = function(babelTypes) {
+  const {types: t} = babelTypes;
+  return {
+    visitor: {
+      VariableDeclarator(path) {
+        const {id, init} = path.node;
+        if (t.isIdentifier(id, {name: 'IS_DEV'})
+            && t.isBooleanLiteral(init, {value: true})) {
+          path.replaceWith(
```

Comment:
Just replace the init.

Reply:
I actually tried doing that...
can you please explain why line 73 is still true after this transform 

https://astexplorer.net/#/gist/e39e874a4c808c16842a259c850df995/87476cd3629dc264148bd0b40a2daa95b83bce3d

Reply:
https://astexplorer.net/#/gist/e39e874a4c808c16842a259c850df995/b890b7f1668f81110bd8e64b4a4e053f60f67368

Reply:
Replacing the path isn't a bad idea, but if you want to get around the issue, change the following in your snippet.

`const {id, init} = node;` => `let {id, init} = node;`